metrics.md

```diff
@@ -379,6 +379,8 @@ dependencies data are:
 - Package name (e.g., lodash)
 - Package version (e.g., 1.2.3)
 - File path for lockfile (e.g., frontend/yarn.lock)
+- Analysis of external dependency calls. (e.g., from flask import Response, Response(status=204)    )
```

Comment:
A few lines above this, we say that these features power "Dependency Search" and "License Compliance". Perhaps we should come up with a new umbrella term for this new feature we want to add, which is neither. Perhaps "Dependency Intelligence"?

Wording wise, perhaps you could say something like "Usage of external dependencies (e.g. `Response` in `from flask import Response`), and maybe add another bullet underneath this reprising the comment that `Source code is NOT collected`